tools/flow/README.md

```diff
@@ -0,0 +1,21 @@
+# Setting things up.
+
+Flow doesn't try to recreate the key creation parts of what Linaro CA
```

Comment:
LITE Bootstrap Server now (https://github.com/Linaro/lite_bootstrap_server)

Reply:
Fixed this.